README.md

```diff
@@ -21,65 +21,25 @@ This Action can be used on its own, but we recommend using it with [MetaMask/act
 In order for this action to run, the project must have a [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)-compatible changelog, even if it's empty.
 You should use [`@metamask/auto-changelog`](https://github.com/MetaMask/auto-changelog) to do this.
 
-Add the below workflow file to your repository in the path `.github/workflows/create-release-pr.yml`.
+You must add the [Create Release Pull Request Workflow](#create-pull-request-workflow) (or something equivalent) to use this action.
+Depending on the review processes of your organization, you may also want to add the [Require Additional Reviewer Workflow](#require-additional-reviewer-workflow).
+
+### Create Release Pull Request Workflow
+
+Add the workflow file referenced below to your repository in the path `.github/workflows/create-release-pr.yml`.
 You'll notice that the workflow is manually triggered using the `workflow_dispatch` event.
 Once you've added the workflow file, you trigger the workflow via the Actions tab of the [GitHub Web UI](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) or the [`gh` CLI](https://cli.github.com/manual/gh_workflow_run).
 
-```yaml
```

Comment:
Hmm. Kinda nice to have this here, so it's easy to copy+paste. Not a huge deal though.

Reply:
It's kinda nice, but I removed it because I've at various point forgotten to update it in the README when I've changed the actual file.